nixos/lib/test-driver/test-driver.py

```diff
@@ -986,18 +994,52 @@ def subtest(name: str) -> Iterator[None]:
         help="re-use a VM state coming from a previous run",
         action="store_true",
     )
-    (cli_args, vm_scripts) = arg_parser.parse_known_args()
+    arg_parser.add_argument(
+        "-I",
+        "--interactive",
+        help="drop into a python repl and run the tests interactively",
+        action="store_true",
+    )
+    arg_parser.add_argument(
+        "--start-scripts",
+        metavar="START-SCRIPT",
+        action=EnvDefault,
+        envvar="startScripts",
+        nargs="*",
+        help="start scripts for participating virtual machines",
+    )
+    arg_parser.add_argument(
+        "--vlans",
+        metavat="VLAN",
```

Comment:
did you test the code? i just tried to run a test and got the python parser choke on `metavat`.

Reply:
Damn it, a dumb typo. I'd have hoped the CI test had catched it. Sry.

Reply:
i checked your code out and ran `nix-build nixos/tests/login.nix` and i think the same expression would have been run by the normal CI tests. if not, then i'd be a bit riddled.

Reply:
ok, [fixed](https://github.com/NixOS/nixpkgs/compare/d7d294e9d0cfc13fb54f32e7b6956a6a43589304..1fd7d960bc9d562263e9ed8820925137999e8a18)